### ISSUE TITLE
Need jax and jaxlib modules for last release and those doesn't get installed by default

### ISSUE THREAD

**Issue by Aimnor** [myQLM/myqlm-issues#24]
jax and jaxlib are needed for last release and those doesn't install by default.
Just update your module accordingly.

- First:
```
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_converter_myqlm.py:37: in <module>
    from qat.lang.AQASM import Program, H, X, Y, Z, I, S, T, PH, RX, RY, RZ, SWAP, CNOT, CSIGN, AbstractGate
__init__.py:18: in init qat.lang.__init__
    ???
decorator.py:15: in init qat.lang.decorator
    ???
E   ModuleNotFoundError: No module named 'jax'
```
- Then after jax installation:
```
Hint: make sure your test modules/packages have valid Python names.
Traceback:
venv10/lib/python3.10/site-packages/jax/_src/lib/__init__.py:24: in <module>
    import jaxlib as jaxlib
E   ModuleNotFoundError: No module named 'jaxlib'

The above exception was the direct cause of the following exception:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_converter_myqlm.py:37: in <module>
    from qat.lang.AQASM import Program, H, X, Y, Z, I, S, T, PH, RX, RY, RZ, SWAP, CNOT, CSIGN, AbstractGate
__init__.py:18: in init qat.lang.__init__
    ???
decorator.py:15: in init qat.lang.decorator
    ???
venv10/lib/python3.10/site-packages/jax/__init__.py:35: in <module>
    from jax import config as _config_module
venv10/lib/python3.10/site-packages/jax/config.py:17: in <module>
    from jax._src.config import config  # noqa: F401
venv10/lib/python3.10/site-packages/jax/_src/config.py:27: in <module>
    from jax._src import lib
venv10/lib/python3.10/site-packages/jax/_src/lib/__init__.py:26: in <module>
    raise ModuleNotFoundError(
E   ModuleNotFoundError: jax requires jaxlib to be installed. See https://github.com/google/jax#installation for installation instructions.
```
Fixed after installing jax & jaxlib

**Comment by ArnaudAtos** [comment]
Dear @MarionQuandela,
Thank you very much for your feedback. A new version of myQLM will be released next week, fixing this issue

**Comment by Aimnor** [comment]
Fix with 1.9.3 version
